modules/ROOT/pages/to-share-api-asset-to-portal.adoc

```diff
@@ -4,6 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+An organization's Exchange administrators control whether each API asset created in the organization is shared in the public portal. Set which users have the Exchange administrator role in the Access Management > Roles menu, and find more information in xref:to-set-permissions.adoc[Set User Access Permissions].
```

Comment:
Because it uses imperative voice and uses a menu cascade, this type of wording is better reserved for an actual step (task topic).
This intro. text is sharing, but adding and removing are different things.

Reply:
This led me to look at the View version of this file as a whole, to get a better context.
Although you can publish this as-is, I don't think that the container topic should be titled "Share an asset..." and contain "Add" and "Remove." Sharing refers only to the intro. text; configuring or managing to the second two.
Consider something like Create or Configure or Manage Public Assets, and then follow with the first part of your current intro, including ...:

     Addressing the audience as "you."
     Telling WHY your audience wants or needs to perform these tasks.
     Summarizing the content: that Creating/Configuring/Managing includes giving access or assigning roles,
     adding assets, and removing assets (add under what circumstances).